tests/gem5/gpu/test_gpu_pannotia.py

```diff
@@ -39,14 +39,14 @@
 dataset_path = joinpath(resource_path, "pannotia-datasets")
 
 binary_links = {
-    "bc.gem5": "https://storage.googleapis.com/dist.gem5.org/dist/v24-0/test-progs/pannotia/bc.gem5",
-    "color_max.gem5": "https://storage.googleapis.com/dist.gem5.org/dist/v24-0/test-progs/pannotia/color_max.gem5",
-    "color_maxmin.gem5": "https://storage.googleapis.com/dist.gem5.org/dist/v24-0/test-progs/pannotia/color_maxmin.gem5",
-    "mis_hip.gem5": "https://storage.googleapis.com/dist.gem5.org/dist/v24-0/test-progs/pannotia/mis_hip.gem5",
+    "bc.gem5": "https://gem5dist.blob.core.windows.net/dist/v24-0/test-progs/pannotia/bc.gem5",
+    "color_max.gem5": "https://gem5dist.blob.core.windows.net/dist/v24-0/test-progs/pannotia/color_max.gem5",
+    "color_maxmin.gem5": "https://gem5dist.blob.core.windows.net/dist/v24-0/test-progs/pannotia/color_maxmin.gem5",
+    "mis_hip.gem5": "https://gem5dist.blob.core.windows.net/dist/v24-0/test-progs/pannotia/mis_hip.gem5",
```

Comment:
Is there a argument for "hardcoding" the Azure storage URL? The "dist.gem5.org" domain was specifically created to point to the location of the 'dist' storage. If we utilize "dist.gem5.org" as much as possible, any server migration (such as the on we are doing right now in migrating from Google Cloud to Azure) can be completed by simply updating the 'dist.gem5.org' DNS settings. 

We clearly implemented this for our Google Cloud Bucket instance as well, but I can’t recall a compelling reason for doing so.

```suggestion
    "bc.gem5": "https://dist.gem5.org/dist/v24-0/test-progs/pannotia/bc.gem5",
    "color_max.gem5": "https://dist.gem5.org/dist/v24-0/test-progs/pannotia/color_max.gem5",
    "color_maxmin.gem5": "https://dist.gem5.org/dist/v24-0/test-progs/pannotia/color_maxmin.gem5",
    "mis_hip.gem5": "https://dist.gem5.org5rt/dist/v24-0/test-progs/pannotia/mis_hip.gem5",
```